### DISCUSSION TITLE
How long between approval and enrollment in Github Education?

### DISCUSSION THREAD

**Discussion by maben221** [community/community#62543] [GitHub Education]
### Hi everyone,

I just got approved for the Github Student Pack 2 days ago but I wondered how long does it take to be enrolled in it and be able to use its features?

**Comment by D3Dream** [comment]
4-6 weeks

**Reply by maben221** [reply]
Damn I should've applied a little earlier... thanks for the info!

**Comment by Awrsha** [comment]
Here are the typical next steps after being approved:

You'll receive an email confirmation that your application was approved. This may take 1-2 business days.

GitHub then needs to provision your student benefits, like the freedomains, coupons, and education resources. This can take 1-3 business days.

You'll then receive an email with instructions on how to claim your student benefits. This email contains coupon codes to redeem your free private repositories, Azure credits, etc.

After redeeming your benefits, it may take another 1-2 business days for them to become active on your account and for you to gain full access.

So in total, from the time your application is approved to when you can fully use all the student benefits, it's typically 3-7 business days. The longer end of that range is more common.

To check the status of your student pack:

Log in to your GitHub account.

Go to your settings (the gear icon) > Developer settings.

Under "Student Developer Pack," it will say either "Applied" if still pending or "Active" once your benefits have been provisioned.

If it's been longer than 7-10 days and your pack still says "Applied," you may want to contact GitHub support to verify the status and ensure there are no issues.

Hope this timeline helps you have an idea of what to expect over the next few days as your GitHub Student Developer Pack is set up! Let me know if you have any other questions.

**Reply by maben221** [reply]
Thanks for the detailed reply! I'm looking forward to having the features provisioned into my account.

**Reply by vikrant019** [reply]
![image](https://github.com/community/community/assets/85669352/5bf64bbe-565b-4954-b736-a534329b96d1)
i have received this, how i claim my student benefits

**Reply by Md-Arif-hasnat99** [reply]
i have same siruation like you
i got approved in 30th september
how much time i need to weight to br able to acces it

**Comment by moonquekes** [comment]
Hello, 

May I ask how long you could use the package after receiving the approval? I also met the same situation recently

**Reply by Naynn2003** [reply]
Its been 13-15 days , still not approved!

**Comment by tahatehran** [comment]
I hope that now that there are more packages and gifts, they will do the work faster. I had done it before, but the university didn't accept the time at that time and they didn't accept the application for the master's degree in artificial intelligence. Let's see how many days it will take.

**Reply by tahatehran** [reply]
He knew better than to just use an artificial intelligence model and open source to help, as previously stated, the domain .dev .Now the domain .tech
